app/reducers/index.ts

```diff
@@ -119,6 +122,7 @@ export interface RootState {
   notifications: StateFromReducer<typeof notificationsAccountsProvider>;
   bridge: StateFromReducer<typeof bridgeReducer>;
   banners: BannersState;
+  performance: PerformanceState;
```

Comment:
Should we add isTest flag here ?
```suggestion
  performance: isTest ? PerformanceState : undefined;
```

Reply:
You can assign the base reducers to a variable and conditionally append performance to it depending on the isTest value